### PR TITLE
chore(Storybook): fix story control naming inconsistencies

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-1891)

### CODE REVIEW

**Review by pladaria — APPROVED** [review]

File: src/__stories__/callout-story.tsx [hunk 1]
```diff
@@ -16,19 +16,12 @@ type Args = {
     title: string;
     description: string;
     actions: string;
-    withIcon: boolean;
-    isClosable: boolean;
+    icon: boolean;
```
Comment (yceballost) [inline src/__stories__/callout-story.tsx:19]: should we use "Asset" in this case? We have pending to change "icon" prop to "asset". Maybe we can go ahead with asset in stories? 

https://github.com/Telefonica/mistica-web/issues/626
image

Comment (marcoskolodny) [reply src/__stories__/callout-story.tsx]: Fixed in Callout, Dialog and Feedback

**Review by atabel — APPROVED** [review]

File: src/__stories__/feedback-screen-story.tsx [hunk 2]
```diff
@@ -86,18 +86,18 @@ export const FeedbackScreenStory: StoryComponent<FeedbackScreenArgs> = ({
     multipleParagraphs,
     animateText,
     showIcon,
```
Comment (yceballost) [inline src/__stories__/feedback-screen-story.tsx:88]: icon instead?

**Review by yceballost — APPROVED** [review]

File: src/__stories__/feedback-screen-story.tsx (hunk 2 above)
Comment (marcoskolodny) [reply src/__stories__/feedback-screen-story.tsx]: good catch. Fixed

**Pull request merged**